src/point_one/fusion_engine/messages/control.h

```diff
@@ -377,6 +377,75 @@ struct alignas(4) VersionInfoMessage : public MessagePayload {
   char fw_version_str[0];
 };
 
+/**
+ * @brief Device identifier information, (@ref MessageType::DEVICE_ID, version
+ *        1.0).
+ * @ingroup config_and_ctrl_messages
+ *
+ * This message contains ID data for each of the following, where available:
+ * - HW - A unique ROM identifier pulled from the device HW
+ * - User - A value set by the user to identify a device
+ * - Receiver - A unique ROM identifier pulled from the GNSS receiver
+ *
+ * The message payload specifies the length of each string (in bytes). It is
+ * followed by each of the listed ids consecutively. The values are _not_ null
+ * terminated and aren't necessarily strings. The interpretation of the strings
+ * can be distinguished based on the `device_type`.
```

Comment:
Not totally sure what you mean here? I think this saying "the way each field is populated (strings or binary) depends on the type of device, indicated by @ref device_type" right?